gk/main.c

```diff
@@ -634,6 +711,34 @@ gk_proc(void *arg)
 			process_gk_cmd(gk_cmds[i], instance);
 			mb_free_entry(&instance->mb, gk_cmds[i]);
         	}
+
+		if (ipv6_if_configured(&gk_conf->net->front) && acl.num > 0) {
+			ret = rte_acl_classify(
```

Comment:
Ok, we don't need to do anything, DPDK will pick the most efficient algorithm available automatically.